vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_int8.py

```diff
@@ -136,14 +91,23 @@ def create_weights(self, layer: torch.nn.Module,
                     data=torch.empty(1, dtype=torch.int8),
                     weight_loader=weight_loader)
                 layer.register_parameter("input_zero_point", input_zero_point)
+        else:
```

Comment:
If we're doing static but symmetric, where is the zp being set to None? Or is that not needed